docs/en/install.md

```diff
@@ -104,9 +101,21 @@ These steps will install KubeVela controller and its dependency.
     helm install -n vela-system kubevela kubevela/vela-core --set installCertManager=true
     ```
    
-    If you want to try the latest master branch, try the following command:
+    If you want to try the latest master branch, add flag `--devel` in command `helm search` to choose the latest
```

Comment:
how does the helm distinguish a chart can be normally install or must use `--devel`?  What is the parttern? I guess a version with a suffix will cause it into `--devel`? Am I right?

Reply:
Your guessing is correct. `--devel` means `use development versions (alpha, beta, and release candidate releases), too. Equivalent to version '>0.0.0-0'. If --version is set, this is ignored`.

For my local PoC, all these tags will display with `--devel`  flag.
```
kubevela/vela-core       	0.4.0-rc1.11.g4d8be8c	0.4.0-rc1.11.g4d8be8c	A Helm chart for Kube Vela core
kubevela/vela-core       	0.4.0-latest-master  	0.4.0-latest-master  	A Helm chart for Kube Vela core
kubevela/vela-core       	0.0.3-rc.1           	0.0.3-rc.1           	A Helm chart for Kube Vela core
kubevela/vela-core       	0.0.3-rc             	0.0.3-rc             	A Helm chart for Kube Vela core
```